README.md

```diff
@@ -31,9 +31,29 @@ There are three levels of configuration needed in order for publishing and fetch
    - Specify the store id
    - Choose the docker registry artifact plugin
    - Specify the `Docker Registry Url`
-   - Specify `username` and `password`
+   - Specify the `Registry Type`
+   - Specify registry details
+
+   If `Registry Type` is `ecr`:
+
+   ![](images/artifact_store_ecr.png)
+
+   If `Registry Type` is `other`:
+
+   ![](images/artifact_store_other.png)
+
+###### Artifact Store Properties
+
+| Properties                     | Description                                                              | Required                  |
+| -------------------------------| ------------------------------------------------------------------------ | --------------------------|
+| `RegistryURL`                  | Registry to push and pull images                                         | `true`                    |
```

Comment:
LGTM. Required `true` seems to indicate that the value should be `true`. Perhaps call the column `Mandatory` and set the value to `yes` or `"yes" if type is ...`